### ISSUE TITLE
The documentation website for manuals on v6 doesn't work - error 404

### ISSUE THREAD

**Issue by petrvecera** [sequelize/sequelize#13612]
## Issue Description

Links to manuals no longer work: https://sequelize.org/master/manual/migrations.html
It's still possible to find this with version 5 https://sequelize.org/v5/manual/migrations.html

### Additional context

It had to happen in the last 2 days. As I read those guides without any problems. Google also links to those 404 pages

**Comment by WikiRik** [comment]
I think it might be due to the changes that @sushantdhiman did in https://github.com/sequelize/sequelize.org

**Comment by StanislavNikolov** [comment]
https://web.archive.org/web/20210828205451/https://sequelize.org/master/

**Comment by dianalduran** [comment]
Thank you @StanislavNikolov ! The website was working just fine 15 hours ago therefore the issue is recent.

**Comment by JoshuaDietz** [comment]
Well now the v6 documentation kind of works but it looks really ugly. Probably because the browser tries to load a bunch of scripts that don't exist (anymore?)

**Comment by sushantdhiman** [comment]
Sequelize documentation is built & deployed using https://github.com/sequelize/sequelize.org.

It was not being deploying from last 9 months and thus was outdated. I have migrated it to Github Actions as Travis no longer offers a free plan. Documentation will auto deploy every midnight now.

But it seems `npm run docs` script from `main` branch is currently [not working](https://github.com/sequelize/sequelize.org/runs/4055017374?check_suite_focus=true#step:6:515). I have requested current maintainers to take a look at that script. Once it is fixed documentation will auto deploy itself.

Sorry for this inconvenience :) 

cc @sequelize/maintainers
